internal/clients/connect/client_connect.go

```diff
@@ -179,6 +179,11 @@ type connectGetContentDTO struct {
 	// Owner        *ownerOutputDTO   `json:"owner,omitempty"`
 }
 
+func (c *ConnectClient) ContentDetails(contentID types.ContentID, body *ConnectContent, log logging.Logger) error {
```

Comment:
New client method to pull content data, helps to validate the user has access to it.